### DISCUSSION TITLE
Bienvenue dans l'espace de discussion de guide-R !

### DISCUSSION THREAD

**Discussion by larmarange** [larmarange/guide-R#1] [Annonces]
Vous pouvez poser ici toutes vos questions en lien avec guide-R.

**Comment by LiseCuzin** [comment]
Bonjour
Depuis la mise à jour de mon MacOs avec Ventura 13.4.1 R ne peut plus s'initialiser.
J'ai mis à jour la version, ainsi que XQuartz, rien n'y fait.
Avec vous une idée ?? Je n'ai rien trouvé en ligne, je suis bien embêtée
Merci d'avance
